OnlyR/Services/RecordingDestination/RecordingDestinationService.cs

```diff
@@ -73,12 +73,7 @@ public RecordingCandidate GetRecordingFileCandidate(
 
         private static string GenerateCandidateFilePath(string folder, DateTime dt, int increment, AudioCodec codec)
         {
-            var fileExtension = codec switch
-            {
-                AudioCodec.Mp3 => "mp3",
-                AudioCodec.Wav => "wav",
-                _ => throw new NotSupportedException("Unsupported codec")
-            };
+            var fileExtension = $"{codec.GetExtensionFormat()}";
```

Comment:
No string interpolation needed here

Reply:
Now that's interesting...
I see this here, but I actually removed this line completely from the code. The current state of the code reflects that too. So this line is no longer present

Reply:
Thanks - I see that too. Sorry!

Reply:
I wish this to be the biggest problem in our lives😄

Reply:
I think the project is ready to build. I propose to create a pre-release, referencing the #84 feature request.

Reply:
Alright, let's go.🙂